File: src/main/java/no/nav/veilarboppfolging/controller/response/OppfolgingPeriodeMinimalDTO.java
package no.nav.veilarboppfolging.controller.response;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.ZonedDateTime;
import java.util.UUID;

@Data
@Accessors(chain = true)
public class OppfolgingPeriodeMinimalDTO {
    private UUID uuid;
    private ZonedDateTime startDato;
    private ZonedDateTime sluttDato;
}

File: src/main/kotlin/no/nav/veilarboppfolging/controller/response/OppfolgingPeriodeMinimalDTO.kt
package no.nav.veilarboppfolging.controller.response

import java.time.ZonedDateTime
import java.util.*

class OppfolgingPeriodeMinimalDTO(
    val uuid: UUID,
    val startDato: ZonedDateTime,
    val sluttDato: ZonedDateTime?
)
